docker-compose.yml

```diff
@@ -17,12 +17,14 @@ services:
 
   app:
     # Uncomment this line to use the official program_intent_engagement base image
-    image: openedx/program_intent_engagement
-
+    # image: openedx/program_intent_engagement
```

Comment:
for local development, rather than use a published image, use the local `Dockerfile` and context to be built/used. This is similar to how `edx-enterprise` works, etc.

https://github.com/openedx/edx-enterprise/blob/master/docker-compose.yml#L6-L8